definitions/npm/xorshift_v1.x.x/flow_v0.32.0-/xorshift_v1.x.x.js

```diff
@@ -0,0 +1,18 @@
+declare module 'xorshift' {
+  declare class $npm$xorshift {
```

Comment:
Oh, I was wondering about that. Is there any documentation on what becomes exported/visible?